src/ray/core_worker/core_worker.cc

```diff
@@ -2648,6 +2657,15 @@ Status CoreWorker::ExecuteTask(
     absl::MutexLock lock(&mutex_);
     current_tasks_.emplace(task_spec.TaskId(), task_spec);
     if (resource_ids) {
+      // kBundle_ResourceLabel* is a phantom resource used only for scheduling.
+      // Since the task is already scheduled onto the worker, we don't need it.
+      for (auto iter = resource_ids->begin(); iter != resource_ids->end();) {
+        if (MatchBundleResource(iter->first)) {
+          iter = resource_ids->erase(iter);
```

Comment:
Actually can you check to make sure we won't show this resource in other places as well like `ray status` pending requests etc